packages/webdriverio/src/scripts/isElementInViewport.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * check if element is within the viewport
+ * https://gist.github.com/davidtheclark/5515733#gistcomment-2113205
```

Comment:
Can we add a comment noting that any changes to this file need to be reflected in `isElementClickable`?

Reply:
done